pkg/cmd/cli/cmd/version.go

```diff
@@ -128,38 +128,37 @@ func (o VersionOptions) RunVersion() error {
 			return
 		}
 
-		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
```

Comment:
not anymore.  The only failure was from negotiation - david and I discussed
and it seemed reasonable that if you can't hit /oapi we might as well just
use whatever you passed - subsequent calls will really fail.

On Tue, Sep 6, 2016 at 9:10 PM, Jordan Liggitt <notifications@github.com>
wrote:

> In pkg/cmd/cli/cmd/version.go
> <https://github.com/openshift/origin/pull/10824#discussion_r77746479>:
>
> > @@ -128,38 +128,37 @@ func (o VersionOptions) RunVersion() error {
> >  			return
> >  		}
> >
> > -		ocVersionBody, err := oClient.Get().AbsPath("/version/openshift").Do().Raw()
>
> won't the oClient, kClient, err := o.Clients() call above fail on a
> non-openshift server?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/openshift/origin/pull/10824/files/e2a63d2b900e2b5da5ba40d13651444f35d49e3a#r77746479>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ABG_p6W3M9jgd2KfpCw8jcB_NSmuzmlgks5qng8bgaJpZM4J2Nxd>
> .
>